packages/faceted-search/src/components/BasicSearch/BasicSearch.component.js

```diff
@@ -105,6 +104,9 @@ const BasicSearch = ({
 	const badgeFacetedContextValue = { state, dispatch, onSubmit };
 	// removable = undefined means badge can be removed (backward compatible change)
 	const hasRemovableBadge = state.badges.some(badge => badge.properties.removable !== false);
+	const quickSearchMinLength =
```

Comment:
Quick search input will trigger after either 1 char , or will use the `minLength` definition of quicksearchable badges